Client/src/main/java/org/projectcardboard/client/view/battleview/CardAnimation.java

```diff
@@ -12,6 +14,7 @@
 
 import org.projectcardboard.client.models.gui.ImageLoader;
```

Comment:
have you heard of "Dry"  ?  (don't repeat yourself)

At a quick glance the only difference I can see between the two cases is the path.   In the abstract, the current structure looks like this: 

```
if x:
       path
       logic
if y:
       path
       logic
```

But could be refactored to:

```
path = x ′  if x   y ′  if y
logic
```